dependencies.gradle

```diff
@@ -0,0 +1,25 @@
+// Add your dependencies here
+
+dependencies {
+    file("libs/biomesoplentyAPI.jar")
+    file("libs/CropLoadCore-0.1.4-dev.jar")
+    file("libs/twilightforest-1.7.10-2.3.8.7-dev-deobf.jar")
+    file("libs/GTNewHorizonsCoreMod-1.7.10-1.9.02-dev.jar")
```

Comment:
No need to specify them, ZIP/JAR files in the libs directory are always added as dependency

Reply:
Fair, but I'd like to convert them over to jitpack/maven at some point, don't think they were ready at the time I was doing this so they got manually added.  At least this way they won't (easily) get lost when revisited

Reply:
I would propose to store binary dependencies as files in a folder called `dependencies`. This way you must specify them AND you may apply `compileOnly` wherever possible.

Reply:
But that can be done at migration time